subsys/bluetooth/services/ras/rrsp/ras_rrsp.c

```diff
@@ -205,6 +207,86 @@ static void rd_overwritten_ccc_cfg_changed(struct bt_gatt_attr const *attr, uint
 	LOG_DBG("Ranging Data Overwritten CCCD changed: %u", value);
 }
 
+BT_GATT_SERVICE_DEFINE(
+	rrsp_svc, BT_GATT_PRIMARY_SERVICE(BT_UUID_RANGING_SERVICE),
+	/* RAS Features */
+	BT_GATT_CHARACTERISTIC(BT_UUID_RAS_FEATURES, BT_GATT_CHRC_READ, BT_GATT_PERM_READ_ENCRYPT,
+			       ras_features_read, NULL, NULL),
+	/* On-demand Ranging Data */
+	BT_GATT_CHARACTERISTIC(BT_UUID_RAS_ONDEMAND_RD, BT_GATT_CHRC_INDICATE | BT_GATT_CHRC_NOTIFY,
+			       BT_GATT_PERM_NONE, NULL, NULL, NULL),
+	BT_GATT_CCC_MANAGED(((struct _bt_gatt_ccc[]){BT_GATT_CCC_INITIALIZER(
```

Comment:
will be followed up upstream https://github.com/zephyrproject-rtos/zephyr/pull/88257